.env

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_URL = http://localhost:3000/
```

Comment:
We don't push the .env file, what we do instead is create a .env.dist file and we add just the KEY of the environment variable, not the value.

```javascript
NEXT_PUBLIC_URL=
```